app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt

```diff
@@ -557,7 +551,7 @@ class MainActivity : SimpleActivity(), PhotoProcessor.MediaSavedListener, Camera
 
     private fun hasPhotoModePermissions(): Boolean {
         return if (isTiramisuPlus()) {
-            hasPermission(PERMISSION_READ_MEDIA_IMAGES) && hasPermission(PERMISSION_CAMERA)
+            hasPermission(PERMISSION_READ_MEDIA_IMAGES) && hasPermission(PERMISSION_READ_MEDIA_VIDEO)  && hasPermission(PERMISSION_CAMERA)
```

Comment:
had to add `hasPermission(PERMISSION_READ_MEDIA_VIDEO)` to fix the crash on Android 13+, we have no way of requesting multiple permissions via `handlePermission` API from the `commons` module. 
 
 The system automatically grants `PERMISSION_READ_MEDIA_VIDEO` when the user grants `PERMISSION_READ_MEDIA_IMAGES` without showing a second permission dialog, so it's okay to do this.